components/00-base/01-colors/_02-color-properties.scss

```diff
@@ -0,0 +1,83 @@
+// Create CSS custom properties based on the color-palette map.
+// For each color variable (e.g. `text` and `background`) a css custom
+// property will be created. (e.g. `--c-text:` and `--c-background:`).
+$format: 'hsl';
```

Comment:
See comment above. Default color formatting is `hsl` but this can be changed to `rgb` so custom properties are defined as `rgba`.